plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableDataStoreMeta.php

```diff
@@ -0,0 +1,180 @@
+<?php
+/**
+ * OrdersTableDataStoreMeta class file.
+ */
+
+namespace Automattic\WooCommerce\Internal\DataStores\Orders;
+
+/**
+ * Mimics a WP metadata (i.e. add_metadata(), get_metadata() and friends) implementation using a custom table.
+ */
+class OrdersTableDataStoreMeta {
```

Comment:
Does this needs to be a `Orders`TableDataStoreMeta ? Other than the value returned by `get_db_info` nothing else is really specific, might as well make this class generic now?

Reply:
✅